src/main/scala/examples/Router.scala

```diff
@@ -50,12 +50,14 @@ class Router extends Module {
   val io    = IO(new RouterIO(n))
   val tbl   = Mem(depth, UInt(BigInt(n).bitLength.W))
 
-  when(reset.toBool) {
-    io.read_routing_table_request.nodeq()
-    io.load_routing_table_request.nodeq()
-    io.read_routing_table_response.noenq()
-    io.in.nodeq()
-    io.outs.foreach { out => out.noenq() }
+  io.read_routing_table_request.nodeq()
```

Comment:
How about a comment here that we are relying on last connect semantics to override these as
necessary